frigate/video.py

```diff
@@ -279,7 +283,9 @@ def get_current_frame_with_objects(self):
 
         # draw the bounding boxes on the screen
         for obj in detected_objects:
-            label = "{}: {}%".format(obj['name'],int(obj['score']*100))
+            # since debug mode can show all kinds of objects, we need to calculate the area outside of `add_objects`
+            area = (obj['xmax']-obj['xmin'])*(obj['ymax']-obj['ymin'])
```

Comment:
Since the `area` box is only _truly_ calculated in the `add_objects` class, I needed to recalculate it here in for the debug view when it's showing objects that _are not_ `person`.

I still need to set it to the `obj['area']` because the `best_person` frame (without extra object para extraction) still needs it.

Reply:
Not my favorite since it's kinda already calculated elsewhere. But it works great.